pkgs/servers/web-apps/mealie/default.nix

```
@@ -0,0 +1,123 @@
+{ python310, python310Packages, fetchFromGitHub, mkYarnPackage, lib, ...}: let
+  version = "v1.0.0-RC2";
+  src = fetchFromGitHub {
+    owner = "mealie-recipes";
+    repo = "mealie";
+    rev = version;
+    sha256 = "sha256-/sht8s0Nap6TdYxAqamKj/HGGV21/8eYCuYzpWXRJCE=";
+  };
+
+  frontend = (mkYarnPackage {
+    name = "mealie-frontend";
+    src = "${src}/frontend";
+    packageJSON = "${src}/frontend/package.json";
+    yarnLock = "${src}/frontend/yarn.lock";
+  }).overrideAttrs (old:
+    {
+      doDist = false;
+
+      buildPhase = ''
+        runHook preBuild
+        shopt -s dotglob
+
+        rm deps/${old.pname}/node_modules
+        mkdir deps/${old.pname}/node_modules
+        pushd deps/${old.pname}/node_modules
+        ln -s ../../../node_modules/* .
+        popd
+        export NUXT_TELEMETRY_DISABLED=1
+        yarn --offline --non-interactive --production=false build
+        yarn --offline --non-interactive --produciton=false generate
+        runHook postBuild
+      '';
+
+      installPhase = ''
+        runHook preInstall
+        mv deps/${old.pname}/dist $out
+        runHook postInstall
+      '';
+    });
+
+in python310Packages.buildPythonPackage rec {
+  pname = "mealie";
+  inherit version src;
+  format = "pyproject";
+
+  makeWrapperArgs = [
+    "--set STATIC_FILES ${frontend}"
+  ];
+
+  patches = [
+    ./mealie_statedir.patch
+    ./mealie_init_db.patch
+    ./mealie_logger.patch
+  ];
+
+  nativeBuildInputs = [
+    python310Packages.poetry-core
+  ];
+
+  propagatedBuildInputs = with python310Packages; [
+    jinja2
+    pillow
+    pyyaml
+    sqlalchemy
+    aiofiles
+    # Outdated one in NixOS package repository
+    (alembic.overrideAttrs (old: {
+      src = fetchFromGitHub {
+        owner = "sqlalchemy";
+        repo = "alembic";
+        rev = "rel_1_11_3";
+        sha256 = "sha256-AmkLfrmTu5KLILDBVqLo6on7xtT8hSSnxnNym6dzEq4=";
+      };
+    }))
+    aniso8601
+    appdirs
+    apprise
+    bcrypt
+    extruct
+    fastapi
+    gunicorn
+    lxml
+    orjson
+    passlib
+    psycopg2
+    pyhumps
+    recipe-scrapers
+    uvicorn
+    python-multipart
+    python-slugify
+    rapidfuzz
+    python-dotenv
+    python-jose
+    ldap
+    pytesseract
+  ];
+
+  doCheck = true;
+  checkInputs = with python310Packages; [
+    pytestCheckHook
+  ];
+
+  passthru = {
+    python = python310;
+    pythonPkg = python310Packages;
+    interpreter = "${python310}/bin/python3";
+    pythonPath = python310.pkgs.makePythonPath propagatedBuildInputs;
+    inherit frontend src;
```

```suggestion
    inherit python frontend src;
```

The other attributes _shouldn't_ be necessary, no need to put them in `passthru`.